services/src/test/resources/describe/wdl_1_0/task_inputs_at_workflow_level/description.json

```diff
@@ -1,52 +1,11 @@
 {
-  "valid" : true,
+  "valid" : false,
   "errors" : [
+    "Failed to process workflow definition 'my_workflow' (reason 1 of 1): Failed to process 'call my_task' (reason 1 of 1): To be called as a task it must declare and pass-through the following required values via workflow inputs: task_input"
```

Comment:
We can't break a workflow that does this, people use it and complained when we accidentally broke it before

Maybe for `development`?

https://broadworkbench.atlassian.net/browse/BA-6021
https://github.com/broadinstitute/cromwell/pull/5214

Reply:
@aednichols agreed. Backwards compatibility is the user-friendly thing to do. I will have a look again.
I might start a new branch instead of reverting all the wdl 1.0 test cases.

Reply:
Sounds good. Sorry about this, I know it must suck to write a bunch of code that complies with the spec only to find out this compatibility requirement later